server/auth/lib.py

```diff
@@ -183,8 +215,11 @@ def restrict_access_superadmin(route: Callable):
 
     @functools.wraps(route)
     def wrapper(*args, **kwargs):
-        if not is_superadmin():
+        if not is_superadmin(session):
             raise Forbidden(description="requires superadmin privileges")
+
+        update_last_request_timestamp(session)
```

Comment:
is this happening twice, once in restrict-access and once here?

Reply:
these two decorators cover different sets of routes